swarm/api/http/server.go

```diff
@@ -522,6 +522,12 @@ func (s *Server) HandleGetList(w http.ResponseWriter, r *Request) {
 // HandleGetFile handles a GET request to bzz://<manifest>/<path> and responds
 // with the content of the file at <path> from the given <manifest>
 func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
+	// ensure the root path has a trailing slash so that relative URLs work
+	if r.uri.Path == "" && !strings.HasSuffix(r.URL.Path, "/") {
+		http.Redirect(w, &r.Request, r.URL.Path+"/", http.StatusMovedPermanently)
+		return
+	}
+
```

Comment:
Does this mean that a / is added 
- to every URL? 
- only to the first part of the URL (the 'domain')
- to every URL that references a manifest but not to URLs referencing other content.

I think we are doing the second, but we want the third option, yes?
I want mysite.eth --> mysite.eth/
I want mysite.eth/www --> mysite.eth/www/
I want mysite.eth/www/myimage.png to stay unchanged.